appinfo/info.xml

```diff
@@ -42,8 +42,8 @@ For an demo of this app visit [https://demo.passman.cc](https://demo.passman.cc)
         <database>pgsql</database>
         <database min-version="5.5">mysql</database>
         <lib>openssl</lib>
-        <nextcloud min-version="17" max-version="18"/>
-        <owncloud min-version="17" max-version="18"/>
+        <nextcloud min-version="17" max-version="19"/>
+        <owncloud min-version="17" max-version="19"/>
```

Comment:
Note that the current latest version of owncloud is 10.5.  I bumped both versions for consistency so that it should behave without regression, but this should probably be reviewed as a followup in the future by someone that is more familiar with owncloud.

Reply:
After seeing #660 I went ahead and patched the owncloud version to 10 in a followup commit in this PR.  This should fix the versioning for both.